### DISCUSSION TITLE
Python

### DISCUSSION THREAD

**Discussion by ArminEskandarii** [vpnhood/VpnHood#341] [Ideas]
Hi. Will there be a version in the future that allows us to install vpnhood on the server by python?

**Comment by bobvhood** [comment]
The installation script is a Linux bash script. I am unsure what you mean by Python which is a programming language.

**Reply by ArminEskandarii** [reply]
Sorry I didn't notice your respond earlier. But just like @ylmzfurkan is saying. I mean a python package or module that we can install on the server using the pip command in python. And that should allow us to easily run VPN-HOOD on the server.

**Reply by bobvhood** [reply]
Unfortunately we are not familiar with PHP package/module 😞

**Comment by ylmzfurkan** [comment]
The ability to install VPNhood on a server using Python would depend on several factors, including the design and architecture of the VPNhood software, its compatibility with different deployment environments, and the development roadmap of the VPNhood project.

If the VPNhood team intends to provide a Python-based installation option in the future, it may involve creating a Python package or module that can be easily installed and configured on a server. This could provide a more streamlined and automated deployment process for users.

**Reply by ArminEskandarii** [reply]
Yeah that's exactly what I mean. A python package/module. I hope that comes out soon 🙏❤
